htdocs/comm/action/class/actioncomm.class.php

```diff
@@ -2170,8 +2168,7 @@ public function sendEmailsReminder()
 			$error++;
 		}
 
-		if (!$error)
-		{
+		if (!$error) {
 			// Delete also very old past events (we do not keep more than 1 month record in past)
 			$sql = "DELETE FROM ".MAIN_DB_PREFIX."actioncomm_reminder";
 			$sql .= " WHERE dateremind < '".$this->db->idate($now - (3600 * 24 * 32))."'";
```

Comment:
The variable $actionCommReminder does not seem to be defined for all execution paths leading up to this point.